verifiers/internal/gha/builder.go

```diff
@@ -17,6 +17,9 @@ var (
 	trustedBuilderRepository = "slsa-framework/slsa-github-generator"
 	e2eTestRepository        = "slsa-framework/example-package"
 	certOidcIssuer           = "https://token.actions.githubusercontent.com"
+	// This is used in cosign's CheckOpts for validating the certificate. We
+	// do specific builder verification after this.
+	certSubjectRegexp = "https://github.com/*"
```

Comment:
that will also be necessary for npm provenance verification (the one without the trusted builder)

Reply:
do we intend on verifying non-L3 provenance though?

but yes - it's just a nit that cosign v2 requires some validation on it so I had to add it even though we do a more detail check on it later.